win/pkgs/utf8proc.ps1

```diff
@@ -0,0 +1,69 @@
+#Requires -RunAsAdministrator
+
+Get-Content "$PSScriptRoot/utils/re-entry.ps1" -Raw | Invoke-Expression
+$ErrorActionPreference="Stop"
+
+. "$PSScriptRoot/env/mirror.ps1"
+. "$PSScriptRoot/env/toolchain.ps1"
+
+pushd ${Env:TMP}
+$repo="${Env:GIT_MIRROR}/JuliaStrings/utf8proc.git"
+$proj="$($repo -replace '.*/','' -replace '.git$','')"
+$root="${Env:TMP}/$proj"
+
+rm -Force -Recurse -ErrorAction SilentlyContinue -WarningAction SilentlyContinue "$root"
+if (Test-Path "$root")
+{
+    echo "Failed to remove `"$root`""
+    Exit 1
+}
+
+$latest_ver='v' + $($(git ls-remote --tags "$repo") -match '.*refs/tags/v[0-9\.]*$' -replace '.*refs/tags/v','' | sort {[Version]$_})[-1]
+git clone --depth 1 --single-branch -b "$latest_ver" "$repo"
+pushd "$root"
+mkdir build-win
```

Comment:
Use `build` per CMake convention.
The reason you found `build-win` for some lib is because there's `build` or `Build` (due to Win case insensitivity) dir already.